tests/Integration/Lti/Responder/LtiOutcomeResponderTest.php

```diff
@@ -44,7 +45,7 @@ public function testCreateReplaceResultResponse(): void
         /** @var Environment $twig */
         $twig = static::$container->get(Environment::class);
 
-        $messageIdentifier = 'e36f227c-2946-11e8-b467-0ed5f89f718b';
+        $messageIdentifier = UuidV4::fromString('e36f227c-2946-11e8-b467-0ed5f89f718b');
```

Comment:
Isn't better to work on LtiOutcomeResponder to also use UuidV6 when generating these identifiers?
This way we could have just one version. Wdyt?

Reply:
I think V4 is more suitable here, we don't store these identifiers and we don't want to order them or anything like that, I would prefer to not touch it for now.